### DISCUSSION TITLE
obs.

### DISCUSSION THREAD

**Discussion by gastonoterom** [aio-libs/aiohttp#7332] [Q&A]
nvm.

**Comment by Dreamsorcerer** [comment]
There are examples in the documentation on sending a file, what is or is not working for you?
https://docs.aiohttp.org/en/stable/client_quickstart.html#post-a-multipart-encoded-file

> It appears that the 'files' attribute works differently in the aiohttp client and session objects.

I don't see a files attribute in aiohttp...
